perfkitbenchmarker/benchmark_spec.py

```diff
@@ -155,6 +157,12 @@ def __init__(self, benchmark_module, benchmark_config, benchmark_uid):
         self.config.vm_groups if self.config.relational_db is None else
         relational_db.VmsToBoot(self.config.relational_db.vm_groups))
 
+    self.vpn_service = None
+    self.vpns = {}  # dict of vpn's
+    self.vpn_gws = {}  # dict of vpn gw's
```

Comment:
Please name vpn_gws as vpn_gateways for readability.

Reply:
Fixed.